tests/coalaCITest.py

```diff
@@ -19,63 +19,66 @@ def tearDown(self):
         sys.argv = self.old_argv
 
     def test_log(self):
-        retval, output = execute_coala(
+        retval, stdout, stderr = execute_coala(
```

Comment:
stderr is being ignored here.  either

1. Assert it is empty, either with `stdout_only` or a `self.assert*`
2. or do a proper test of its contents.